packages/flutter/lib/src/services/text_input.dart

```diff
@@ -774,7 +774,7 @@ abstract class TextSelectionDelegate {
   set textEditingValue(TextEditingValue value);
 
   /// Hides the text selection toolbar.
```

Comment:
nit: documentation of the optional parameter.

Reply:
Good call, I think this was a bit confusing overall.  I added a bunch of comments to make this more clear, and I renamed the parameter from handlesOnly to hideHandles.